src/engagement_db_to_analysis/configuration.py

```diff
@@ -8,10 +9,16 @@ class DatasetTypes:
     DEMOGRAPHIC = "demographic"
     RESEARCH_QUESTION_ANSWER = "research_question_answer"
 
+@dataclass
+class AgeCategoryConfiguration:
+    age_analysis_dataset: str
+    categories: dict
+
```

Comment:
```suggestion


```
my editor is very noisy about PEP-8 violations without double-blank lines in modules

Reply:
Updated thanks.